pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_content_checklist.adoc

```diff
@@ -0,0 +1,365 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+
+[id='ref_content_checklist_{context}']
+== Content Checklist for {ProductName}
+
+[role="_abstract"]
+In order to be sure that every component of the {ProductName} tool chain can work with your content, be sure it complies with the following guidelines:
+
+.Assembly requirements
+[cols="30%,35%,35",options="header"]
+|====
+|Requirement
+|Description
+|Example
+
+|Avoid unsupported includes
+
+|The only supported structure for an include is: `include::path/to/some/module/some-module.adoc[leveloffset=+x]`
+
+Always have an empty line between includes.  This applies to both master.adoc and other assemblies.
+|*Supported*
+`include::meta/enterprise/meta/proc_providing-feedback-on-red-hat-documentation.adoc[]`
+
+`include::modules/con_concept-module.adoc[leveloffset=+1]`
+
+`include::assemblies/some_assembly.adoc[leveloffset=+2]`
+
+*Not supported*
+
+`:leveloffset: +1
+modules/core-services/proc_registering-the-system-after-the-installation.adoc[leveloffset: -1]`
```

Comment:
Lisa is correct, the include is missing. Also, the `leveloffset: -1` should be on it's own line
```suggestion
modules/core-services/proc_registering-the-system-after-the-installation.adoc
leveloffset: -1`
```

Reply:
Great --- will fix and resubmit.

Reply:
Like this? Maybe I didn't understand -- it still looks odd to me. 

*Not supported*

`:leveloffset: +1
include::modules/core-services/proc_registering-the-system-after-the-installation.adoc
leveloffset: -1`

Reply:
`:leveloffset: +1 +
include::modules/core-services/proc_registering-the-system-after-the-installation.adoc +
leveloffset: -1`

GitHub is erasing my hard returns (+)

Reply:
> Like this? Maybe I didn't understand -- it still looks odd to me.
> 
> _Not supported_
> 
> `:leveloffset: +1 include::modules/core-services/proc_registering-the-system-after-the-installation.adoc leveloffset: -1`

In asciidoc, if you have a bunch of includes with no leveloffset like `include::some-module.adoc[]` and put them in between `:leveloffset: +1` and `:leveloffset: -1`, they will all inherit `leveloffset=+1`, as if you had all of the included formatted like this `include::modules/core-services/proc_registering-the-system-after-the-installation.adoc[leveloffset:=+1]`. Unfortunately it’s not supported as far as we know.